src/form/ChestAPI/session/network/handler/ClosurePlayerNetworkHandler.php

```
@@ -0,0 +1,22 @@
+use pocketmine\form\ChestAPI\session\network\NetworkStackLatencyEntry;
+
+final class ClosurePlayerNetworkHandler implements PlayerNetworkHandler{
+
+	/**
+	 * @param Closure(Closure) : NetworkStackLatencyEntry $creator
+	 */
+	public function __construct(
+		readonly private Closure $creator
+	){}
+
+	public function createNetworkStackLatencyEntry(Closure $then) : NetworkStackLatencyEntry{
```

PHPStan (2.1.15)

Method pocketmine\form\ChestAPI\session\network\handler\ClosurePlayerNetworkHandler::createNetworkStackLatencyEntry() has parameter $then with no signature specified for Closure.


(missingType.callable)